packaging/cloudify-amqp-influx.spec

```diff
@@ -0,0 +1,67 @@
+
+Name:           cloudify-amqp-influx
+Version:        %{CLOUDIFY_VERSION}
+Release:        %{CLOUDIFY_PACKAGE_RELEASE}%{?dist}
+Summary:        Cloudify's REST Service
+Group:          Applications/Multimedia
+License:        Apache 2.0
+URL:            https://github.com/cloudify-cosmo/cloudify-amqp-influxdb
+Vendor:         Gigaspaces Inc.
+Packager:       Gigaspaces Inc.
+
+BuildRequires:  python >= 2.7, python-virtualenv
+Requires:       python >= 2.7, influxdb
+Requires(pre):  shadow-utils
+
+
+%define _venv /opt/amqpinflux/env
+
+
+%description
+Cloudify's Broker pulls Cloudify formatted Metrics from RabbitMQ and posts them in InfluxDB.
+
+
+%prep
+
+
+%build
+
+virtualenv /opt/amqpinflux/env
+
+%_venv/bin/pip install --upgrade pip setuptools
+%_venv/bin/pip install --upgrade "${RPM_SOURCE_DIR}/"
+
+
+%install
+
+mkdir -p %{buildroot}/opt/amqpinflux
+mv %_venv %{buildroot}/opt/amqpinflux
+
+
+# Create the log dir
+mkdir -p %{buildroot}/var/log/cloudify/influxdb
+
+# Copy static files into place. In order to have files in /packaging/files
+# actually included in the RPM, they must have an entry in the %files
+# section of this spec file.
+cp -R ${RPM_SOURCE_DIR}/packaging/files/* %{buildroot}
+
+
+%pre
+
+groupadd -fr amqpinflux
+getent passwd amqpinflux >/dev/null || useradd -r -g amqpinflux -d /opt/amqpinflux -s /sbin/nologin amqpinflux
```

Comment:
this doesn't need cfyuser anywhere? what about reading the ca cert?

Reply:
The behaviour is copied from https://github.com/cloudify-cosmo/cloudify-manager-install/pull/98/files#diff-073ab67cc715e5ab82d7764a190f7fa4L58 which doesn't seem to use `cfyuser`